### PR TITLE
Add tracked time fix to doctor (part of #11111)

### PR DESCRIPTION
**Pull request by 6543**
Backports the tracked-time fix from #11111 (part of #11111)

Fixes tracked time errors following repo deletion (#10280)  and adds the fix to the default doctor tasks !

### CODE REVIEW

File: cmd/doctor.go
Comment (6543) [inline cmd/doctor.go]:
```suggestion
		isDefault: true,
```

**Review by zeripath — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]
For consistency, I'd rather front-port this PR and work #11111 from there.

**Review by zeripath — CHANGES_REQUESTED** [review]
Blocked until #11111 is merged

**Comment by 6543** [comment]
since doctor got released this is now **kind/breaking** !

**Comment by lafriks** [comment]
Released in what version?

**Comment by 6543** [comment]
1.11 since doctor got backported by @zeripath

**Review by lafriks — APPROVED** [review]

**Comment by zeripath** [comment]
Do we still want this backported to 1.11.6? my suspicion is not but I guess it might be helpful?

**Comment by 6543** [comment]
@zeripath I would partialy backport it to fix an issue described in #10280

I would leave this implementation as it is now and the comand wont change from there on only get more functions&checks :)

**Comment by 6543** [comment]
@zeripath or should I add DB version check and move fix functions into models ?

(as 11111 has now)

**Review by zeripath — APPROVED** [review]

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**
